yt/data_objects/static_output.py

```diff
@@ -9,6 +9,7 @@
 from stat import ST_CTIME
 
 import numpy as np
+from unyt.exceptions import UnitConversionError, UnitParseError
```

Comment:
something to note (not necessaary for this PR I think) `UnitParseError` can also be imported from yt.units.unit_object  and is imported that way in some places and this way in others.

Reply:
hopefully this will eventually be harmonised once we settle for either (#2597 ?)